devices/AtModem/AtModem.nfproj

```diff
@@ -0,0 +1,293 @@
+    </Compile>
+    <Compile Include="Modem\ModemBase.cs" />
+    <Compile Include="Modem\ModemResponse.cs" />
+    <Compile Include="Modem\Sim7070.cs" />
+    <Compile Include="Modem\Sim7080.cs" />
+    <Compile Include="Modem\Sim7090.cs" />
+    <Compile Include="Modem\Sim800.cs" />
+    <Compile Include="Mqtt\Sim7080MqttClient.cs" />
+    <Compile Include="Network\INetwork.cs">
+      <ExcludeFromStyleCop>true</ExcludeFromStyleCop>
+    </Compile>
+    <Compile Include="Network\Sim7080Network.cs" />
+    <Compile Include="Network\Sim800Network.cs" />
+    <Compile Include="Http\Http\X509Certificate.cs" />
+    <Compile Include="PDU\Pdu.cs" />
+    <Compile Include="PDU\PduMessage.cs" />
+    <Compile Include="PDU\PduType.cs" />
+    <Compile Include="PDU\ReadOnlySpanExtensions.cs" />
+    <Compile Include="PDU\SmsDeliver.cs" />
+    <Compile Include="PDU\SmsSubmit.cs" />
+    <Compile Include="PDU\SpanChar.cs">
+      <ExcludeFromStyleCop>true</ExcludeFromStyleCop>
+    </Compile>
+    <Compile Include="DTOs\SimCardInformation.cs" />
+    <Compile Include="DTOs\NetworkInformation.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Sms\ISmsProvider.cs">
+      <ExcludeFromStyleCop>true</ExcludeFromStyleCop>
+    </Compile>
+    <Compile Include="Sms\GenericSmsProvider.cs" />
+    <Compile Include="DTOs\SystemMode.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="mscorlib">
+      <HintPath>packages\nanoFramework.CoreLibrary.1.14.2\lib\mscorlib.dll</HintPath>
+    </Reference>
+    <Reference Include="nanoFramework.M2Mqtt.Core">
+      <HintPath>packages\nanoFramework.M2Mqtt.Core.5.1.92\lib\nanoFramework.M2Mqtt.Core.dll</HintPath>
+    </Reference>
+    <Reference Include="nanoFramework.Runtime.Events">
+      <HintPath>packages\nanoFramework.Runtime.Events.1.11.6\lib\nanoFramework.Runtime.Events.dll</HintPath>
+    </Reference>
+    <Reference Include="nanoFramework.Runtime.Native">
+      <HintPath>packages\nanoFramework.Runtime.Native.1.6.6\lib\nanoFramework.Runtime.Native.dll</HintPath>
+    </Reference>
+    <Reference Include="nanoFramework.System.Collections">
+      <HintPath>packages\nanoFramework.System.Collections.1.5.18\lib\nanoFramework.System.Collections.dll</HintPath>
+    </Reference>
+    <Reference Include="nanoFramework.System.Text">
+      <HintPath>packages\nanoFramework.System.Text.1.2.37\lib\nanoFramework.System.Text.dll</HintPath>
+    </Reference>
+    <Reference Include="System.IO.Ports">
+      <HintPath>packages\nanoFramework.System.IO.Ports.1.1.60\lib\System.IO.Ports.dll</HintPath>
+    </Reference>
+    <Reference Include="System.IO.Streams">
+      <HintPath>packages\nanoFramework.System.IO.Streams.1.1.38\lib\System.IO.Streams.dll</HintPath>
+    </Reference>
+    <Reference Include="System.Math">
+      <HintPath>packages\nanoFramework.System.Math.1.5.29\lib\System.Math.dll</HintPath>
+    </Reference>
+    <Reference Include="System.Net">
```

Comment:
Sys.Net shouldn't be referenced... right?

Reply:
correct, good catch! Removed!